.yamato/config/universal_lighting.metafile

```diff
@@ -127,9 +127,10 @@ platforms:
         flavor: b1.xlarge
         model: rtx2080
       editmode:
-        type: Unity::VM
+        type: Unity::VM::GPU
         image: sdet/gamecode_win10:stable
-        flavor: b1.large
+        flavor: b1.xlarge
+        model: rtx2080
```

Comment:
The xlarge flavor is unfortunately not available for GPUs, you will have to use the b1.large one https://internaldocs.hq.unity3d.com/bokken/reference/flavors/

Reply:
Ok will do